internal/bgp/bgp.go

```diff
@@ -671,3 +691,28 @@ func buildTCPMD5Sig(addr net.IP, key string) tcpmd5sig {
 
 	return t
 }
+
+// localAddressExists returns true if the address addr exists on any of the
+// network interfaces in the ifs slice. The function doesn't return errors and
+// instead returns false if an error has occurred while looking for the
```

Comment:
This isn't true anymore, right? If an error happens, we ignore it and continue with the rest.

Maybe it is best to remove this sentence entirely? Since the return value is bool (not error) and is quite of a short function to read if needed